cmd/policy/policy.go

```diff
@@ -134,9 +134,62 @@ func NewCommand(config *settings.Config, preRunE validator) *cobra.Command {
 		return cmd
 	}()
 
+	update := func() *cobra.Command {
+		var policyPath string
+		var policyID string
+		var updateRequest policy.UpdateRequest
+
+		cmd := &cobra.Command{
+			Short: "Update a policy",
+			Use:   "update",
+			RunE: func(cmd *cobra.Command, args []string) error {
+				policyData, err := os.ReadFile(policyPath)
+				if err != nil {
+					return fmt.Errorf("failed to read policy file: %w", err)
+				}
+
+				updateRequest.Content = string(policyData)
+
+				client := policy.NewClient(*policyBaseURL, config)
+
+				result, err := client.UpdatePolicy(*ownerID, policyID, updateRequest)
+				if err != nil {
+					return fmt.Errorf("failed to update policy: %w", err)
+				}
+
+				enc := json.NewEncoder(cmd.OutOrStdout())
+				enc.SetIndent("", "  ")
+
+				if err := enc.Encode(result); err != nil {
+					return fmt.Errorf("failed to encode result to stdout: %w", err)
+				}
+
+				return nil
+			},
+			Args:    cobra.ExactArgs(0),
+			Example: `policy update --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --policy-id e9e300d1-5bab-4704-b610-addbd6e03b0b --name policy_name --active --policy ./policy.rego`,
+		}
+
+		cmd.Flags().StringVar(&policyID, "policy-id", "", "id of the policy to update")
+		cmd.Flags().StringVar(&updateRequest.Name, "name", "", "name of policy to update")
+		cmd.Flags().StringVar(&updateRequest.Context, "context", "config", "policy context")
+		// Default value of bool flag must be true or false (cannot be nil).
+		// To avoid accidently enabling a policy, will make this flag required.
+		cmd.Flags().BoolVarP(&updateRequest.Active, "active", "a", true, "is policy active (to ensure a policy is inactive, use --active=false)")
+		cmd.Flags().StringVar(&policyPath, "policy", "", "path to rego policy file")
+
+		cmd.MarkFlagRequired("policy-id")
```

Comment:
I'm not sure if it's best to make all flags mandatory. Say a user once created a policy long back (they don't have content for policy with them anymore locally), and they want to set the policy to active, they would just want to update the active status.

Reply:
Technically none are mandatory! You could be smart and in the API code check if the UpdateRequest is empty and do a noop